en/guides/students/getting-started/index.md

```diff
@@ -1,52 +1,27 @@
 ---
-title: "[nl] Van start met Dodona als student"
-description: "Tutorial: van start met Dodona als student"
+title: "Getting Started with Dodona as a Student"
+description: "Tutorial: getting started with Dodona as a student"
 order: 1
 ---
 
-::: warning Sorry
-For now, this page is only available in Dutch. Sorry!
-:::
+# Getting Started with Dodona as a Student
+On this page, you will find all the information you need to get started with Dodona as a student.
 
-# Van start met Dodona als student
+## What is Dodona?
 
-> Op deze pagina vind je alle informatie om vlot van start te gaan met Dodona als student.
+Dodona is an online practice platform for learning programming. The web application was developed at Ghent University to support practical sessions for large groups of students. Dodona focuses primarily on the automatic testing and feedback for submitted solutions to programming exercises. Currently, Dodona supports the programming languages Python, JavaScript, Java, Kotlin, C#, bash, Prolog, Haskell, and R.
```

Comment:
```suggestion
Dodona is an online exercise platform for learning to code. The web application was developed at Ghent University to support practical sessions for large groups of students. Dodona focuses primarily on the automatic testing and feedback for submitted solutions to programming exercises. Currently, Dodona supports the programming languages Python, JavaScript, Java, Kotlin, C#, bash, Prolog, Haskell, and R.
```

The list of languages is not up to date. At least sql and html are missing. Maybe even more through tested?